src/app/src/main/java/com/cmput301f21t09/budgetprojectname/services/database/DatabaseService.java

```diff
@@ -0,0 +1,17 @@
+package com.cmput301f21t09.budgetprojectname.services.database;
+
+import com.google.firebase.firestore.FirebaseFirestore;
+
```

Comment:
Add documentation and specify this is a singleton design pattern

Reply:
This should never be a singleton.  Firebase Firestore is connected to the application context and therefore any instance of it should never be stored in global scope.